CMakeLists.txt

```diff
@@ -18,6 +21,12 @@ project(${APP_NAME}
 )
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
+if(ANDROID)
+  # You probably have installed Qt for Android somewhere else
+  # other than /usr. In that case, this is necessary
+
+  LIST(APPEND CMAKE_FIND_ROOT_PATH "/")
```

Comment:
This looks suspiciously like a fix for the local setup and not a general config that belongs into `if(ANDROID)`.
I think that should be moved to the documentation to be passed as command line option instead.